test/scripts/test-ciphers-from-pod.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# $SERVER is expected to be provided as an environment variable
+# and should represent the location of the console via internal
+# IP address, represented by pod.status.podIP, and including :8443.
+# example: 10.128.0.30:8443
+echo "connecting to ${SERVER}"
+
+
+# ensure we ignore weak ciphers
+# CBC (cipher block chaining) are no longer reliable and should not be used
+# CBC ciphers use an IV (initialization vector) and a chaining mechanism.
+# The chaining mechanism means that a single bit error in a ciphertext block
+# will invalidate all previous blocks.  The chaining was good in that it hides
+# plaintext patterns, but is inferior to other cipher modes.
+INVALID_CIPHER_SAMPLE=(
+  DES-CBC3-SHA                  # from QE
+  RSA-AES-128-CBC-SHA256
+  ECDHE-RSA-3DES-EDE-CBC-SHA    # disabled to mitigate SWEET32 attack
+  RSA-3DES-EDE-CBC-SHA          # disabled to mitigate SWEET32 attack
+)
+
+for CIPHER in "${INVALID_CIPHER_SAMPLE[@]}"
+do
+  RESULT=$(openssl s_client -connect "${SERVER}" -cipher "${CIPHER}" 2>&1)
+  if [[ $? -eq 0 ]]
+  then
+    echo "invalid cipher suite used to connect to console (${CIPHER})"
+    exit 1
+  else
+    echo "invalid cipher was correctly denied (${CIPHER})"
+  fi
+done
+
+
+# CIPHER=ECDHE-ECDSA-CHACHA20-POLY1305 # DENIED
```

Comment:
However, these are also denied :)
Working on figuring out what is up with this.

Reply:
@spadgett curious if you have a thought.